### PR TITLE
[DEV APPROVED] [9569] Add meta-tags for articles and documents

### PR DESCRIPTION
**Pull request by MarcoBgn**
[TP Card](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNzEyRUZDMzlENEJERUE4QjJBRDVEQzdENUE1MEE2OEIifQ==&boardPopup=userstory/9569/silent)

- In order to have SEO, meta-tags have been added to articles
  and documents. Editors can add `meta_description` and `meta_title`
  from the CMS and have it present on each page.
  The meta-tags Gem has been used

- Some VCR cassettes had to be re-recorded and content amended due
  to outdated specs

### CODE REVIEW

File: app/views/lifestages/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <% present(resource) do |template| %>
+  <% set_meta_tags(
+      title: template.meta_title,
+      description: template.description
+    )
+  %>
```
Comment (tomas-stefano) [inline app/views/lifestages/show.html.erb]: Wdyt? <%= template.render_meta_tags %>

File: app/presenters/article_template_presenter.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 class ArticleTemplatePresenter < TemplatePresenter
   include SharedComponents::Strategy
   include SharedComponents::Teaser
+
+  delegate :meta_description, to: :article
+  delegate :meta_title, to: :article
```
Comment (tomas-stefano) [inline app/presenters/article_template_presenter.rb]: We can change to `delegate :meta_title, :meta_description, to: :article`

File: app/presenters/evidence_summary_presenter.rb [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # rubocop:disable Metrics/ClassLength
 class EvidenceSummaryPresenter < BasePresenter
+  delegate :meta_description, to: :document
+  delegate :meta_title, to: :document
```
Comment (tomas-stefano) [inline app/presenters/evidence_summary_presenter.rb]: Same as above

File: spec/presenters/evidence_summary_presenter_spec.rb [hunk 4]
```diff
@@ -6,6 +6,36 @@
...
+
+        expect(document)
+          .to receive(:meta_title)
+
+        presenter.meta_title
+      end
+    end
+
+    context '#meta_description' do
+      it do
```
Comment (tomas-stefano) [inline spec/presenters/evidence_summary_presenter_spec.rb]: Can we add a test description, please?

File: spec/presenters/evidence_summary_presenter_spec.rb [hunk 5]
```diff
@@ -6,6 +6,36 @@
   subject(:presenter) { described_class.new(evidence_summary, view) }
   let(:attributes) { {} }
 
+  describe 'delegated methods' do
+    let(:document) { double('Mas::Cms::EvidenceSummary') }
+
+    context '#meta_title' do
+      it do
```
Comment (tomas-stefano) [inline spec/presenters/evidence_summary_presenter_spec.rb]: Can we add a test description, please?

File: app/presenters/article_template_presenter.rb (hunk 2 above)
Comment (MarcoBgn) [reply app/presenters/article_template_presenter.rb]: Good point 👍

**Comment by MarcoBgn** [comment]
Comments addressed, thanks @tomas-stefano 👍

File: features/article_page.feature [hunk 6]
```diff
@@ -17,15 +17,16 @@ Feature: Article page
...
+      | UK detailed strategy                                     | UK_detailed_strategy.pdf                         |
+      | Financial Capability Progress Report 2017                | Financial_Capability_Progress_Report_2017.pdf   |
     And I should see the call to action box containing the links
       | text                             | link                         |
       | Evidence Hub                     | /general_info                |
       | Evaluation Toolkit               | /common-evaluation-toolkit   |
       | The Steering Groups              | /steering-groups             |
-      | 2015 Financial Capability Survey | /financial-capability-survey |
     And I should see the feedback box with the email "email@moneyadviceservice.org.uk"
+
```
Comment (your) [inline features/article_page.feature]: Any reason for this newline? I can see that you did it consistently in the next files so just wondering why?

Comment (MarcoBgn) [reply features/article_page.feature]: Yes, I thought it might help to visually separate the 'meta tags' context

**Comment by MarcoBgn** [comment]
Thanks for the review @your

**Pull request merged**